### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by jcreedcmu**
Update CHANGELOG.md for 1.0.6 release.

### CODE REVIEW

File: extensions/ql-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## 1.0.6
+## 1.0.6 - 28 February 2020
 
 - Add command to restart query server.
+- Improve cli upgrade versioning.
```
Comment (henrymercer) [inline extensions/ql-vscode/CHANGELOG.md]: Maybe something like:

```suggestion
- Enables support for future versions of the CodeQL CLI.
```

to explain what's changed?

Comment (adityasharad) [reply extensions/ql-vscode/CHANGELOG.md]: Future major/minor versions, perhaps.

File: extensions/ql-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## 1.0.6
+## 1.0.6 - 28 February 2020
 
 - Add command to restart query server.
+- Enables support for future minor upgrades to the CodeQL CLI.
```
Comment (henrymercer) [inline extensions/ql-vscode/CHANGELOG.md]:
```suggestion
- Enable support for future minor upgrades to the CodeQL CLI.
```

File: extensions/ql-vscode/CHANGELOG.md (hunk 1 above)
Comment (jcreedcmu) [reply extensions/ql-vscode/CHANGELOG.md]: I'm confused about this comment; it specifically allows minor-version upgrades, but still mins the major version number.

Comment (adityasharad) [reply extensions/ql-vscode/CHANGELOG.md]: Ah that should have said just 'minor'.

**Pull request merged**